app/src/lib/stores/app-store.ts

```diff
@@ -159,36 +163,33 @@ export class AppStore {
   private currentPullRequestUpdater: PullRequestUpdater | null = null
 
   private repositoryState = new Map<string, IRepositoryState>()
-  private showWelcomeFlow = false
-
+  private _showWelcomeFlow = false
```

Comment:
In one word, consistency. Not married to it though.

Reply:
Yeah, I see the `_` here as being non-standard so that it stands out and make the reader feel uncomfortable.

Can we undo these changes here to reduce the churn as part of this PR?